main.c

```diff
@@ -52,8 +52,7 @@ int main(void)
 
     /*Handle LVGL tasks*/
     while(1) {
-        lv_timer_handler();
-        usleep(5000);
+        usleep(lv_timer_handler() * 1000);
```

Comment:
Good idea, this is just make it easier to understand
```suggestion
        uint32_t idle_time = lv_timer_handler(); /*Returns the time to the next timer execution*/
        usleep(idle_time * 1000);
```